build/Helix/PrepareHelixPayload.ps1

```diff
@@ -55,14 +55,7 @@ Copy-Item "build\helix\runtests.cmd" $payloadDir
 Copy-Item "build\helix\InstallTestAppDependencies.ps1" "$payloadDir"
 Copy-Item "build\Helix\EnsureMachineState.ps1" "$payloadDir"
 
-# Copy the APPX package from the 'drop' artifact dir and Windows Kits
-Copy-Item "$repoDirectory\Artifacts\$ArtifactName\appx\CascadiaPackage_0.0.1.0_$Platform.msix" $payloadDir\CascadiaPackage.zip
-Copy-Item "C:\program files (x86)\Microsoft SDKs\Windows Kits\10\ExtensionSDKs\Microsoft.VCLibs.Desktop\14.0\Appx\Retail\x64\Microsoft.VCLibs.x64.14.00.Desktop.appx" $payloadDir\VCLibs.zip
-
-# Rename it to extension of ZIP because Expand-Archive is real sassy on the build machines
-# and refuses to unzip it because of its file extension while on a desktop, it just
-# does the job without complaining.
-
-# Extract the APPX package
-Expand-Archive -LiteralPath $payloadDir\CascadiaPackage.zip -DestinationPath $payloadDir\appx
-Expand-Archive -LiteralPath $payloadDir\VCLibs.zip -DestinationPath $payloadDir\appx -Force
+# Extract the unpackaged distribution of Windows Terminal to the payload directory,
+# where it will create a subdirectory named terminal-0.0.1.0
+# This is referenced in TerminalApp.cs later as part of the test harness.
+& tar -x -v -f "$repoDirectory\Artifacts\$ArtifactName\unpackaged\WindowsTerminalDev_0.0.1.0_x64.zip" -C "$payloadDir"
```

Comment:
I think --full options are better for CI scripts for clarity.

Reply:
ah so, _the canonical options for tar are the short ones_. windows tar doesn't support the GNU-style modern `--long-options` :)